examples/ansible/setup_neuvector.yml

```diff
@@ -0,0 +1,113 @@
+# and https://documentation.suse.com/alp/micro/html/alp-micro/available-alp-workloads.html#task-run-neuvector-with-podman
+# The playbook supports two operations: setup and removal of NeuVector.
+# The operation is determined by the 'neuvector_state' variable, which can be set to 'present' or 'absent'.
+# When 'neuvector_state' is set to 'present', the playbook will:
+#   - Check if the tools for the NeuVector workload are already installed
+#   - Set SELinux into permissive mode
+#   - Retrieve the NeuVector image
+#   - Execute nevector runlabel INSTALL
+#   - Start the NeuVector service
+#   - Enable the NeuVector service to start on a reboot
+#   - Print a message with the connection details for NeuVector
+# When 'neuvector_state' is set to 'absent', the playbook will:
+#   - Stop the NeuVector service
+#   - Execute nevector runlabel UNINSTALL
+#   - Remove the NeuVector image
+
+- name: Running the NeuVector workload
+  hosts: alphost
+  vars:
+    neuvector_state: present
+    workload:
+      name: neuvector
+      image: registry.opensuse.org/suse/alp/workloads/bci_containerfiles/suse/alp/workloads/neuvector-demo:latest
+
+  tasks:
+    - name: Setup NeuVector
+      when:
+        - neuvector_state == "present"
+      block:
+        - name: Check if workload {{ workload.name }} is already installed
+          become: true
+          containers.podman.podman_container_info:
+            name: "{{ workload.name }}-allinone"
+          register: workload_info
+          changed_when: false
+
+        - name: Set SELinux into permissive mode
+          become: true
+          ansible.posix.selinux:
+            policy: targeted
+            state: permissive
+          when:
+            - workload_info.containers | length == 0
+
+        - name: Retrieve image for workload {{ workload.name }}
+          become: true
+          containers.podman.podman_image:
+            name: "{{ workload.image }}"
+            state: present
+          register: retrieve_workload_image
+          when:
+            - workload_info.containers | length == 0
+
+        - name: Execute nevector runlabel INSTALL
+          become: true
+          ansible.builtin.command: >-
+            podman container runlabel install {{ workload.image }}
+          register: workload_runlabel_install
+          changed_when:
+            - ('already exist' not in workload_runlabel_install.stdout)
+            - workload_info.containers | length == 0
+
+        - name: Start NeuVector service
+          become: true
+          ansible.builtin.systemd:
+            name: neuvector.service
+            state: started
+          register: workload_service_start
+
+        - name: enable service to start on a reboot
+          become: true
+          ansible.builtin.systemd:
+            name: neuvector.service
+            enabled: yes
+          when:
+            - workload_service_start is success
```

Comment:
This is not needed. If the `service start` above will fail the run will die.

Reply:
Yes, that's right I will remove the redundant condition.